drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -8542,7 +8542,7 @@ void RenderingDeviceVulkan::_free_rids(T &p_owner, const char *p_type) {
 		} else {
 			WARN_PRINT(vformat("%d RIDs of type \"%s\" were leaked.", owned.size(), p_type));
 		}
-		for (RID E : owned) {
+		for (const RID &E : owned) {
```

Comment:
Turned all `for (RID foo : bar)` into `for (const RID &foo : bar)`.

I didn't double check the implications but @reduz also seemed to think it's fine.

Reply:
Yeah, const reference is better here.